.github/workflows/build.yml

```diff
@@ -1,40 +0,0 @@
-name: Build
```

Comment:
We probably want to reintroduce this file at some point in the future,
once we add a test for generating the docs.
Removing it should be good for now though.

Reply:
Agree